src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/AsyncWriteProxyEx.cs

```diff
@@ -162,26 +168,25 @@ protected override bool AroundReceive(Receive receive, object message)
         /// <returns>TBD</returns>
         protected override Task<IImmutableList<Exception>> WriteMessagesAsync(IEnumerable<AtomicWrite> messages)
         {
+            var trueMsgs = messages.ToArray();
```

Comment:
work-around for serialization problems caused by the MNTR Cluster.Sharding specs that use this journal impl.